### DISCUSSION TITLE
Can you use KorGE in conjunction with native Android features (like Room, Dagger/Hilt, Jetpack Compose, etc)?

### DISCUSSION THREAD

**Discussion by Dan-Carlin** [korlibs/korge#1194] [Q&A] [answered]
I'm currently building a productivity app that makes use of quite a few Android specific libraries, but I'm finding that in order to pull it off I'm lacking some functionality that could only be provided by a game engine. I looked into LibGDX, but that doesn't seem to play very nice with native Android libraries. Is KorGE a good solution if I'm still planning on using the Android specific stuff?

**Comment by soywiz** [comment, marked as answer]
KorGE can be used just as an android library. You can create an android view or activity with korge:

https://github.com/korlibs/korge-samples/blob/5a259513363945d6edcc5909154e43b484b0b89e/integrations/android/app/src/main/java/org/korge/korgeviewexample/MainActivity.kt

Since it is a library, you can use it in conjuntion with anything you need. In Jetpack Compose Android you can embed custom Android views, so you can also put the KorGE view there.
